### DISCUSSION TITLE
Enable a canonical URL in frontmatter

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress/core#1081] [Q&A] [answered]
Hello,

It is a simple idea: using frontmatter attribute, it'd be nice to add the [meta link "canonical"](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls?hl=fr#rel-canonical-link-method) on the generated html.

I haven't tried to do it, but what are your thoughts on that?

Or has it been done already? I've searched, but I may have missed it.

Cheers!

**Comment by Mister-Hope** [comment, marked as answer]
You can do it with `frontmatter.head`  and there is a plugin `vuepress-plugin-seo2`

**Comment by JeremieLitzler** [comment]
This is what I was looking for! Thanks @Mister-Hope !
